src/store/modules/nft.js

```diff
@@ -84,6 +88,7 @@ const getters = {
       (acc, val) => acc + val.length,
       0
     ),
+  getNFTMetadataById: state => id => state.metadataByNFTIdMap[id],
```

Comment:
class id + nft id is the unique key, so we need to pass class id too

Reply:
Updated